screwdriver.yaml

```diff
@@ -20,14 +20,12 @@ jobs:
             - setup-ci: git clone https://gist.github.com/3d2388b2a7ba658cdcdaffa8cd874e50.git ci
             - get: go get -t ./...
             - gofmt: "find . -name '*.go' | xargs gofmt -s -w"
-            - build: go build -a -o sd-step
-            - get-bzip2: apt-get update && apt-get install -y --no-install-recommends bzip2
             - tag: ./ci/git-tag.sh
-            - release: ./ci/git-release.sh
+            - release: |
+                rm -f ./VERSION
```

Comment:
The `VERSION` file is created by https://gist.github.com/stjohnjohnson/3d2388b2a7ba658cdcdaffa8cd874e50#file-git-tag-sh-L17. 
If the file exists, goreleaser command fails with following error.
```
% goreleaser --skip-publish
   • running goreleaser dev
   • loading config file       file=.goreleaser.yml
   • SETTING DEFAULTS
   • GETTING AND VALIDATING GIT STATE
   ⨯ release failed            error=git is currently in a dirty state:
?? VERSION
```
( This error is not occur when `--skip-validate` is specified )

And because current archive format is `binary` (not archive ), the `files` section is ignored 😢 
```
archive:
  format: binary
```